### ISSUE TITLE
Configurable colours?

### ISSUE THREAD

**Issue by tanekere** [ElianCodes/brutal#59]
Hello 👋,
I would like to change the default colours if this. For example, the i want the orange, pink, whites, etc to look different but there doesn't seem to be any way to do that. I feel like this will suit my website better if I could configure it with [gruvbox](https://github.com/morhetz/gruvbox) colours. Also making the website dark theme would be possible.

It would be cool if you could add an option to make that configurable. Or if that's a bad idea, how i can do so myself in such a way that i will not miss out on any updates to this project. Help is much appreciated! 

In case this is dumb, feel free to tell me why. I am no pro.

**Comment by ElianCodes** [comment]
Yes, there is definitely a lot of explaining to improve. I'm planning on spending some time in the near-future to update the library and write a dedicated documentation.
Thanks for raising the issue here!

**Comment by at-the-vr** [comment]
brutal components (accurately `brutal-ui` components) can use a `color` prop in which you can mention your custom shade, this is repetitive as you can see down the path but its customizable. Screenshot added for the example
![image](https://github.com/ElianCodes/brutal/assets/88548999/30479b0c-6acd-4d8c-8f87-891e0a2b4a19)

**Comment by win8linux** [comment]
Following up from https://github.com/ElianCodes/brutal/issues/59, perhaps after dark mode support is implemented what comes next is being able to define the colour palettes of light and dark modes separately?

**Comment by absurditiesmediallc** [comment]
Maybe I'm missing something about how `astro.config.mjs` functions, but according to your README, another colors arrray can be defined in `astro.config`, but in the `brutal-ui` components for Card, Pill, and Button, it looks like you're pulling in the array directly from a `colors.json` in the `brutal-ui` module. If I'm comprehending this correctly, this isn't actually implemented? There's a couple things here I'm just beginning to wrap my head around, so this is confusing me a lot. Any feedback here will probably get me over the last part of the learning curve, so any comments will have my sincere gratitude.

**Comment by at-the-vr** [comment]
I think your correct @absurditiesmediallc , one way i could make this work was replicate the exact logic from `brutal-ui` that is create a new `colors.json` and have it generate random color strings for the components
